packages/devtools-source-map/src/index.js

```diff
@@ -1,29 +1,23 @@
 // @flow
 
-const { getValue } = require("devtools-config");
 const {
   originalToGeneratedId,
   generatedToOriginalId,
   isGeneratedId,
   isOriginalId,
-  workerTask,
 } = require("./util");
 
+const { workerTask } = require("devtools-modules");
+
 import type { Location } from "devtools-client-adapters/src/types";
 
 let sourceMapWorker;
-function restartWorker() {
-  if (sourceMapWorker) {
-    sourceMapWorker.terminate();
-  }
-  sourceMapWorker = new Worker(getValue("sourceMapWorkerURL"));
+function startWorker(url: string) {
```

Comment:
Can we pair the names? Either start and stop or init and destroy. The current start and destroy seems strange to me.